src/diffusers/models/resnet.py

```diff
@@ -587,6 +588,206 @@ def forward(self, hidden_states):
         return hidden_states * torch.tanh(torch.nn.functional.softplus(hidden_states))
 
 
+class InflatedConv3d(nn.Conv2d):
```

Comment:
Missing docstrings.

Reply:
Not a big fan of inheriting from PyTorch modules - can we try to directly use `nn.Conv2D` below and just add the extra reshaping code each time?

Reply:
Addressed. No more InflatedConv3D. Replaced with pytorch ops